File: Two Pointers/Is subsequence/ans.cpp
class Solution {
public:
    bool isSubsequence(string s, string t) {
        int i=0;
        int j=0;
        int count=0;
        while(i<s.length() && j<t.length()){
            if (s[i]==t[j]){
                i++;
                j++;
                count++;


            }
            else{
                j++;
            }
        }

        return count==s.length();
        
    }
};

File: Array&string/Length of last word/ans.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        n=len(s)
        i=n-1
        count=0
        while i>=0 and s[i]==' ':
            i-=1
        while i>=0 and s[i]!=" ":
            count+=1
            i-=1
        return count